lib/jobs/install-os.js

```diff
@@ -129,6 +129,23 @@ function installOsJobFactory(
                 }
             });
         }
+
+        if (this.options.installPartitions) {
+            _.forEach(this.options.installPartitions, function(partition) {
+                assert.string(partition.mountPoint, "mountPoint must be a string");
+                assert.string(partition.size, "size must be a string");
```

Comment:
Seem need an extra assertion if `size` is not "auto", then it should be a number string.